v3/src/data-interactive/handlers/attribute-handler.ts

```diff
@@ -46,10 +52,39 @@ export const diAttributeHandler: DIHandler = {
     }
     return attributeNotFoundResult
   },
-  create: diNotImplementedYet,
+  create(resources: DIResources, values?: DIValues) {
+    const { dataContext } = resources
+    if (!dataContext) return dataContextNotFoundResult
+
+    // Wrap single attribute in array and bail if any new attributes are missing names
+    const attributeValues = Array.isArray(values) ? values : [values]
+    const attributeErrors = attributeValues.map(singleValue => {
+      if (!singleValue || singleValue.name == null) {
```

Comment:
```suggestion
      if (!singleValue?.name) {
```
An empty string is not a valid name, either.